docs/usermanual.md

```diff
@@ -98,17 +98,130 @@ hello
 
 then the resulting attribute sent to ContextBroker:
 
-{
-   ...
-   "attrHex": {
-     "value": "68656c6c6f"
-     "type": "<the one used at provisiong time for attrHex attribute>"
-   }
-}
+{ ... "attrHex": { "value": "68656c6c6f" "type": "<the one used at provisiong time for attrHex attribute>" } }
+
+Note that every group of 2 character (I.E, the first group, `68`) corresponds to a single ASCII character or byte
+received in the payload (in this case, the value `0x68` corresponds to `h` in ASCII). You can use one of the multiple
+tools available online like [this one](https://string-functions.com/string-hex.aspx)
+
+##### NGSI-v2 and NGSI-LD Measure reporting
+
+It is possible report as a measure a NGSI-v2 or NGSI-LD payload when related device/group is configured with payloadType
+`ngsiv2` or `ngsild`. In these cases payload is ingested as measure where entity attributes are measure attributes and
+`id` and `type` are ignored, since `id` and `type` from device/group configuration provisioned are used, as well as
+`actionType`.
+
+Examples of these `ngsi-v2` payloads are:
```

Comment:
```suggestion
Examples of these `ngsiv2` payloads are (NGSI-v2 batch update format):
```

Reply:
I think better lo align documentation with the exact values that can be used in payloadType (which doesn't use the "-").